test/functionalTest/cases/1316_create_subscription/post_subscriptions.test

```diff
@@ -228,8 +228,8 @@ Date: REGEX(.*)
     "id": "REGEX([0-9a-f]{24})",
     "notification": {
         "attrs": [
-            "humidity",
-            "temperature"
+            "temperature",
```

Comment:
Not sure about the possible problem here.
The attributes come out in the same order as they were sent in the request (and it's not alphabetical order)